### PR TITLE
search & Filter improvements for Figma File search

### PR DESCRIPTION
**Pull request by GLaDO8**
## Description
- You can now search for projects. (using grid.item keywords)
- The dropdown filter now lists all projects inside a team even when multiple teams are configured.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Thanks for your contribution @GLaDO8 🔥 

Do you want to check this @michaelwschultz

**Comment by pernielsentikaer** [comment]
Do you have any comments @michaelwschultz

File: extensions/figma-files/src/index.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { resolveAllFiles } from "./components/fetchFigmaData";
 import { useEffect, useState } from "react";
 import { useCachedPromise } from "@raycast/utils";
 import { getPreferenceValues } from "@raycast/api";
+import { TeamFiles } from "./types";
```
Comment (michaelwschultz) [inline extensions/figma-files/src/index.tsx:9]: Nit - This should probably be imported as a type specifically.

File: extensions/figma-files/src/index.tsx [hunk 2]
```diff
@@ -69,18 +80,26 @@ export default function Command() {
       tooltip={teamID.length > 1 ? "Teams" : "Projects"}
       defaultValue="All"
       onChange={handleDropdownChange}
-      storeValue
+      storeValue={false}
```
Comment (michaelwschultz) [inline extensions/figma-files/src/index.tsx:83]: Could you just remove this prop?

**Review by michaelwschultz — APPROVED** [review]
Thanks again @GLaDO8!

**Review by pernielsentikaer — APPROVED** [review]
Thanks for the review @michaelwschultz, and thanks @GLaDO8 for the fixes 🔥

**Pull request merged**